ams/server/src/test/java/com/netease/arctic/server/manager/TestSpiPluginMananger.java

```diff
@@ -0,0 +1,90 @@
+package com.netease.arctic.server.manager;
+
+import com.netease.arctic.ams.api.AmoroPlugin;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+
+public class TestSpiPluginMananger {
+
+  private SpiPluginManager<TestPlugin> pluginManager;
+  private static final String pluginName1 = "plugin1";
+  private static final String pluginName2 = "plugin2";
+
+  @BeforeEach
+  public void setup() {
+    pluginManager = new SpiPluginManager<TestPlugin>() {};
+  }
+
+  @AfterEach
+  public void tearDown() {
+    pluginManager.close();
+  }
+
+  @Test
+  public void testInstall() {
+    pluginManager.install("plugin1");
```

Comment:
The variable name `pluginName1` should be used instead of a string.

Reply:
That String and this String is not belonged to the module